docker_containers/mindmeld_docker/conf/elasticsearch.yml

```diff
@@ -66,6 +66,9 @@ path.logs: /elasticsearch/log
 # The default list of hosts is ["127.0.0.1", "[::1]"]
 #
 #discovery.zen.ping.unicast.hosts: ["host1", "host2"]
+discovery.type: single-node
```

Comment:
Would this elasticsearch config be used only for development purposes?  Or is this yml file compatible for production deploys as well? 

I guess I'm wondering if the error is really an issue because it seems like one node became healthy before the other and was just waiting for its heartbeat.

Reply:
Good question. I'm not that familiar with how Elasticsearch works. But from this [answer](https://stackoverflow.com/a/61547317),  it seems you're right that this setting does bypass the production checks. I just assumed that people would run their own version of ES instead of the docker container version. Should I change it back to the commented version? That seems to fit the mentioned guidelines too.

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/discovery-settings.html

Reply:
@chrisliu156: does adding an additional node cause the docker instance to be significantly larger?

Reply:
So every elasticsearch cluster has admin nodes and data nodes.  Admin nodes keep track of shard allocation of indexes and general health monitoring while data nodes literally just store the data.  In this single node case that one node acts as the admin and data node.

If you were to increase the amount of nodes you would need three, standard election quorum, to determine which node is the admin node.  So it would be x3 the resources of that one node, of course memory allocation can always be adjusted on a per node basis.

@minhtuev  I think it would be practical to have two config files that the user can choose for deployment.  One for dev purposes and one for production deployments.

Reply:
Got it, can you provide an example of the production config here?

Reply:
Resolved in discussion offline and leaving as is.